frontend/src/component/feature/StrategyTypes/FlexibleStrategy/FlexibleStrategy.tsx

```diff
@@ -123,6 +126,35 @@ const FlexibleStrategy = ({
                     onChange={e => onUpdate('groupId')(e.target.value)}
                     data-testid={FLEXIBLE_STRATEGY_GROUP_ID}
                 />
+                <br />
+                <ConditionallyRender
+                    condition={Boolean(uiConfig?.flags?.strategyVariant)}
+                    show={
+                        <>
```

Comment:
Good for now, probably should be a separate component later on.

Reply:
This is temporary and design will change. Just wanted to demo this POC